spec/index.html

```diff
@@ -427,14 +498,26 @@ <h2>RDF Documents and Syntaxes</h2>
     <em>RDF 1.2 Concepts and Abstract Syntax</em>,
     but can conform to such other specifications that normatively
     reference terms defined here.</p>
+
+  <p>This specification establishes two conformance levels:</p>
```

Comment:
The WG has not agreed that nor the words used. It's WIP.

"Profiles" has also been mentioned.

Also - "This specification" would be the semantics.

At a minimum this text needs a note that it is tentative.
As concepts does not need this language, and the dfn's are unstable, removing it for now would be better.

Reply:
See note added below. My opinion is that if it relates to the data model/abstract syntax, it should be described in Concepts rather than Semantics. I suggested elsewhere that the text could just be removed from this PR and taken up another time. But, the notion that there may be some variation allowed or supported is probably worth signaling sooner rather than later.